collatz_test.py

```diff
@@ -10,8 +10,8 @@ def test_collatz_0bad():
     assert collatz(0) != 1
 
 def test_collatz_general():
-    lengths = [0, 1, 7, 2, 5, 8, 16, 3, 19, 6, 14, 9, 9, 17, 17, 4, 12, 20, 20,
+    lengths = [0, 0, 1, 7, 2, 5, 8, 16, 3, 19, 6, 14, 9, 9, 17, 17, 4, 12, 20, 20,
```

Comment:
No, it's actually correct. You'll notice I changed it from `range(1, ...)` to just `range(...)`, so this will now test to ensure that the zero case works.